src/services/users/useInfiniteUser.tsx

```diff
@@ -57,8 +56,8 @@ export const useInfiniteUsers = (
         signal,
         query: {
           pageSize: 25,
-          page: pageParam,
           ...query,
+          page: pageParam,
```

Comment:
Moving this so that even if the user manually sets the page number in the url, the infinite query will not use it.